lib/web/server.cpp

```diff
@@ -586,6 +646,7 @@ void Socialite::Web::Server::handleConnection(NetStream* stream) {
 		}
 		else if(currentHost->allowDomains.size() == 1 && currentHost->allowDomains[0] == "*") {
 			prepHeader += "Access-Control-Allow-Origin: *\r\n";
+			prepHeader += "Access-Control-Allow-Headers: *\r\n";
```

Comment:
Seems a bit loose, but this is given that this was already defined by the `AllowedDomains` in the config file.